pyuvsim/utils.py

```diff
@@ -75,3 +76,65 @@ def cirs_to_tee_ra(cirs_ra, time):
     gast = time.sidereal_time('apparent', longitude=0)
     tee_ra = cirs_ra + (gast - theta_earth)
     return tee_ra
+
+
+def altaz_to_zenithangle_azimuth(altitude, azimuth):
+    """
+    Convert from astropy altaz convention to UVBeam az/za convention.
+
+    Args:
+        altitude: in radians
+        azimuth: in radians in astropy convention: East of North (N=0, E=90 degrees)
+
+    Returns:
+        zenith_angle in radians
+        azimuth in radians in uvbeam convention: North of East(East=0, North=90 degrees)
+    """
+    input_alt = np.array(altitude)
+    input_az = np.array(azimuth)
+    if input_alt.size != input_az.size:
+        raise ValueError('number of altitude and azimuth values must match.')
+
+    zenith_angle = np.pi / 2 - input_alt
+    new_azimuth = np.pi / 2 - input_az
+
+    if new_azimuth.size > 1:
+        wh_neg = np.where(new_azimuth < -1e-9)
+        if wh_neg[0].size > 0:
+            new_azimuth[wh_neg] = new_azimuth[wh_neg] + np.pi * 2
+    else:
+        if new_azimuth < -1e-9:
+            new_azimuth = new_azimuth + np.pi * 2
+
+    return zenith_angle, new_azimuth
+
+
+def zenithangle_azimuth_to_altaz(zenith_angle, azimuth):
+    """
+    Convert from astropy altaz convention to UVBeam az/za convention.
+
+    Args:
+        zenith_angle: in radians
+        azimuth: in radians in uvbeam convention: North of East(East=0, North=90)
+
+    Returns:
+        altitude in radians
+        azimuth in radians in astropy convention: East of North (N=0, E=90 degrees)
```

Comment:
Notes in degrees, units in radians.

Returns:
        altitude in radians
        azimuth in radians in astropy convention: East of North (N=0, E=pi/2 degrees)

Reply:
fixed.